analysis/druidrestoration/src/modules/features/HotCountGraph.tsx

```diff
@@ -0,0 +1,79 @@
+import SPELLS from 'common/SPELLS';
+import COVENANTS from 'game/shadowlands/COVENANTS';
+import Events from 'parser/core/Events';
+import { Options } from 'parser/core/Module';
+import BuffCountGraph, { GraphedSpellSpec } from 'parser/shared/modules/BuffCountGraph';
+import Panel from 'parser/ui/Panel';
+
+import ConvokeSpiritsResto from '../../modules/shadowlands/covenants/ConvokeSpiritsResto';
+
+const CONVOKE_SPEC_NAME = 'Convoke';
+const CONVOKE_WITH_FLOURISH_SPEC_NAME = 'Convoke w/ Flourish';
+
+class HotCountGraph extends BuffCountGraph {
+  static dependencies = {
+    convokeSpirits: ConvokeSpiritsResto,
+  };
+  convokeSpirits!: ConvokeSpiritsResto;
+  // custom specs to show a different color for Convokes that do and don't proc Flourish
+  convokeSpec!: GraphedSpellSpec;
+  convokeWithFlourishSpec!: GraphedSpellSpec;
+
+  constructor(options: Options) {
+    super(options);
+    if (this.selectedCombatant.hasCovenant(COVENANTS.NIGHT_FAE.id)) {
+      this.addEventListener(Events.fightend, this.onFightEndConvokeCount);
+    }
+  }
+
+  onFightEndConvokeCount() {
+    this.convokeSpirits.convokeTracker.forEach((cast) => {
+      if (cast.spellIdToCasts[SPELLS.FLOURISH_TALENT.id]) {
+        this.addRuleLine(CONVOKE_WITH_FLOURISH_SPEC_NAME, cast.timestamp);
+      } else {
+        this.addRuleLine(CONVOKE_SPEC_NAME, cast.timestamp);
+      }
+    });
+  }
+
+  buffSpecs(): GraphedSpellSpec[] {
+    const buffSpecs: GraphedSpellSpec[] = [];
+    buffSpecs.push({
+      spells: [SPELLS.REJUVENATION, SPELLS.REJUVENATION_GERMINATION],
+      color: '#900090',
+    });
+    buffSpecs.push({ spells: SPELLS.WILD_GROWTH, color: '#00b000' });
+    this.selectedCombatant.hasTalent(SPELLS.CENARION_WARD_TALENT) &&
```

Comment:
I might be insane but an If statement would probably be easier to read here... More LoC but we don't play golf

Reply:
Yeah you're probably right